Code/GraphMol/Descriptors/Crippen.h

```diff
@@ -82,6 +82,23 @@ class CrippenParams {
   ~CrippenParams();
 };
 
+const std::string CrippenClogPVersion = crippenVersion;
+//! calculate the default Wildman-Crippen LogP for a molecule
+/*!
+  See calcCrippenDescriptors
+    \param mol        the molecule of interest
+ */
+double calcClogP(const ROMol &mol);
```

Comment:
I just noticed the name of this. I'm going to rename it to calcLogP/CrippenLogP after merging.
"ClogP" is the thing that the biobyte code generates.